libs/pika/async_mpi/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ endif()
 # Default location is $PIKA_ROOT/libs/mpi/include
 set(async_mpi_headers
     pika/async_mpi/mpi_exception.hpp pika/async_mpi/mpi_polling.hpp
-    pika/async_mpi/transform_mpi.hpp
+    pika/async_mpi/mpi_throttling.hpp pika/async_mpi/transform_mpi.hpp
```

Comment:
Header doesn't exist.

Reply:
oops. I started creating a new file with the throttling code in, but removed it's contents during my conflict merge and forgot to remove the header includes. Will fix.